yt_dlp/extractor/ard.py

```diff
@@ -601,3 +602,186 @@ def fetch_page(page_num):
+                'thumbnail': 'https://api.ardmediathek.de/image-service/images/urn:ard:image:b9b4f1e8b93da4dd',
+                'description': 'md5:3552d571e1959754cff66c1da6c0fdae',
+                'display_id': 'urn:ard:section:855c7a53dac72e0a',
+                'timestamp': 1735629900,
+                'series': 'Auf der Spur – Die ARD Ermittlerkrimis',
+                'channel': 'ARD',
+                'episode': 'Episode 1',
+                'episode_number': 1,
+                'abr': 192,
+                'acodec': 'AAC',
+            },
+        },
+        {
+            'url': 'https://www.ardaudiothek.de/episode/urn:ard:extra:d2fe7303d2dcbf5d/',
+            'info_dict': {
+                'id': 'urn:ard:extra:d2fe7303d2dcbf5d',
+                'ext': 'mp3',
+                'title': 'Trailer: Fanta Vier Forever, Baby!?!',
+                'description': 'md5:b64a586f2e976b8bb5ea0a79dbd8751c',
+                'channel': 'SWR',
+                'duration': 62,
+                'thumbnail': 'https://api.ardmediathek.de/image-service/images/urn:ard:image:48d3c255969be803',
+                'series': 'Fanta Vier Forever, Baby!?!',
+                'timestamp': 1732108217,
+                'upload_date': '20241120',
+                'abr': 128,
+                'acodec': 'MP3',
+            },
+        },
+    ]
+
+    _QUERY_ITEM = '''\
+    query($id: ID!) {
+        item(id: $id) {
+            audioList {
+                href
+                distributionType
+                audioBitrate
+                audioCodec
+            }
+            show {
+              title
+            }
+            image {
+              url1X1
+            }
+            programSet {
+              publicationService {
+                organizationName
+              }
+            }
+            description
+            title
+            duration
+            startDate
+            episodeNumber
+        }
+    }'''
+
+    def _real_extract(self, url):
+        urn = self._match_valid_url(url).group('id')
+        item = self._graphql_query(urn, self._QUERY_ITEM)['item']
+        return {
+            'formats': traverse_obj(item, (
+                'audioList', lambda _, v: url_or_none(v['href']), {
+                    'url': 'href',
+                    'format_id': ('distributionType', {str}),
+                    'abr': ('audioBitrate', {int}),
+                    'acodec': ('audioCodec', {str}),
+                })),
+            'id': urn,
+            **traverse_obj(item, {
+                'channel': ('programSet', 'publicationService', 'organizationName', {str}),
+                'description': ('description', {str}),
+                'duration': ('duration', {int_or_none}),
+                'series': ('show', 'title', {str}),
+                'episode_number': ('episodeNumber', {int_or_none}),
+                'thumbnail': ('image', 'url1X1', {lambda v: v.split('?')[0] if isinstance(v, str) else None}),
```

Comment:
use `update_url` from `..utils`
```suggestion
                'thumbnail': ('image', 'url1X1', {url_or_none}, {update_url(query=None)}),
```